features/step_definitions/api-token.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const jwt = require('jsonwebtoken');
+const request = require('../support/request');
+const testTokenName = 'an access token';
+
+module.exports = function server() {
+    this.When(/^a new API token is generated$/, () =>
+        // Make sure there won't be a conflict: delete the token if it's already there
```

Comment:
In cucumber it looks like `Before` is more of a `beforeEach` and runs before each scenario. The other scenarios are going to want the token to exist, whereas this one does not.

Maybe I should modify the feature file, to say something like "When calvin has no token named 'foo', and generates a token named 'foo'" so that the cleanup can be done in the bit with "has no token named foo"